src/home-assistant.html

```diff
@@ -50,70 +51,70 @@
   }
 };
 
-Polymer({
-  is: 'home-assistant',
-
-  hostAttributes: {
-    icons: null,
-  },
-
-  properties: {
-    connectionPromise: {
-      type: Object,
-      value: window.hassConnection || null,
-      observer: 'handleConnectionPromise',
-    },
-    connection: {
-      type: Object,
-      value: null,
-      observer: 'connectionChanged',
-    },
-    hass: {
-      type: Object,
-      value: null,
-    },
-    icons: {
-      type: String,
-    },
-    iconsLoaded: {
-      type: Boolean,
-      value: false,
-    },
-    showMain: {
-      type: Boolean,
-      computed: 'computeShowMain(hass, iconsLoaded)',
-    },
-  },
-
-  listeners: {
-    settheme: 'setTheme',
-  },
-
-  computeShowMain: function (hass, iconsLoaded) {
+class HomeAssistant extends Polymer.Element {
```

Comment:
Do we want to give these classes a generic name for easy copy-paste, since it's only referenced at the end of the script?

Reply:
Let's keep it to be the name of the element. The class name might show up when we are digging through prototype chains when debugging.